diff --git a/node_modules/plotly.js/dist/plotly.js b/node_modules/plotly.js/dist/plotly.js
index 28f5612..5b9956d 100644
--- a/node_modules/plotly.js/dist/plotly.js
+++ b/node_modules/plotly.js/dist/plotly.js
@@ -34384,6 +34384,9 @@ var layoutUIControlPatterns = [{
   pattern: /^(shapes|annotations)/,
   attr: 'editrevision'
 }, {
+  pattern: /^selections/, 
+  attr: 'selectionrevision'
+},{
   pattern: /^title\.text$/,
   attr: 'editrevision'
 }];
diff --git a/node_modules/plotly.js/src/plot_api/plot_api.js b/node_modules/plotly.js/src/plot_api/plot_api.js
index 75e98f5..5c38053 100644
--- a/node_modules/plotly.js/src/plot_api/plot_api.js
+++ b/node_modules/plotly.js/src/plot_api/plot_api.js
@@ -2394,6 +2394,7 @@ var layoutUIControlPatterns = [
 
     {pattern: /^legend\.(x|y)$/, attr: 'editrevision'},
     {pattern: /^(shapes|annotations)/, attr: 'editrevision'},
+    {pattern: /^selections/, attr: 'selectionrevision'},
     {pattern: /^title\.text$/, attr: 'editrevision'}
 ];
 
